core/transport/src/main/proto/grpc/block_master.proto

```diff
@@ -246,6 +257,20 @@ message GetRegisterLeasePResponse {
   optional GetRegisterLeasePOptions options = 4;
 }
 
+message WorkerPreRegisterInfo {
+  optional PreRegisterCommandType preRegisterCommandType = 1;
+  optional string clusterId = 2;
+  optional int64 workerId = 3;
+}
+
+enum PreRegisterCommandType {
+  UNKNOWN = 0;
+  ACK_REGISTER = 1;
+  REGISTER_PERSIST_CLUSTERID = 2;
+  REGISTER_CLEAN_BLOCKS = 3;
```

Comment:
```suggestion
  REGISTER_FORMAT_BLOCKS = 3;
```
"format" is better?

Reply:
This is different from `formatWorker` but removes Block and cleans up some related data(metrics, BlockMasterSync.report, ...)